tests/fabric/test_peer.py

```diff
@@ -352,7 +373,7 @@ def test_create_channel(
         ]
         mock_get_pod.side_effect = [mock_pod0_ex, mock_pod1_ex]
         create_channel(self.OPTS)
-        mock_random.choice.assert_called_once_with(self.OPTS["orderers"]["names"])
+        mock_random.choice.assert_called()
```

Comment:
Ideally, let's be as specific as possible. Is this called once? If input arguments are complicated, and we call more than once, we can follow this up with an:

    assert mock_random.choice.call_count == 2